cmd/babylond/cmd/root.go

```diff
@@ -130,11 +132,46 @@ func NewRootCmd() *cobra.Command {
 		TextualCoinMetadataQueryFn: authtxconfig.NewGRPCCoinMetadataQueryFn(initClientCtx),
 	}
 
+	EnhanceRootCommandWithoutTxStaking(autoCliOpts, rootCmd)
+
+	return rootCmd
+}
+
+// EnhanceRootCommandWithoutTxStaking excludes staking tx commands
+func EnhanceRootCommandWithoutTxStaking(autoCliOpts autocli.AppOptions, rootCmd *cobra.Command) {
```

Comment:
since we don't have the `x/staking` msg server routes, we still add the staking commands and later manually finds to remove it

It is done this way to avoid need to overwrite `x/staking` app module